templates/cli/lib/parser.js.twig

```diff
@@ -8,7 +8,7 @@ const cliConfig = {
   json: false
 };
 
-const parse = (data) => {
+const parse = (data, child = false) => {
```

Comment:
I don't think using child like this is the right approach because it doesn't work for multiple levels deep.

Reply:
Fixed